packages/flutter_tools/lib/src/aot.dart

```diff
@@ -214,7 +214,7 @@ class AotBuilder {
     final FlutterProject flutterProject = FlutterProject.current();
     final Target target = buildMode == BuildMode.profile
       ? const AotAssemblyProfile()
-      : const AotAssemblyRelease();
```

Comment:
When is AotAssemblyRelease being built with this change?

Reply:
I don't really like how it is set up right now. previously I created "bundle" targets that did nothing but depend on all of the individual parts - I can move towards that setup so that changes like this are less surprising